pytorch_blade/pytorch_blade/quantization/pybind_functions.cpp

```diff
@@ -57,16 +62,153 @@ void remove_placeholder(Module& model) {
   }
 }
 
+void replace_aten_fake_quant_with_custom_version(Module& model) {
+  auto g = model.get_method("forward").graph();
+  // the graph should be inlined first
+  Inline(*g);
+  Symbol sym = Symbol::fromQualString(
+      torch::blade::quantization::custom_fake_quant_name);
+  std::vector<Node*> aten_fake_quant_node;
+
+  for (auto n : g->nodes()) {
```

Comment:
nit: To get a non-const iter, we can always use `auto&&` for simpla. But I guess it's ok to just get a copy of Node iter or Node* here.

Reply:
done